app/models/poll.rb

```diff
@@ -9,8 +10,10 @@ class Poll < ActiveRecord::Base
   has_many :officer_assignments, through: :booth_assignments
   has_many :officers, through: :officer_assignments
   has_many :questions
+  has_many :comments, as: :commentable
 
   has_and_belongs_to_many :geozones
+  belongs_to :author, -> { with_hidden }, class_name: 'User', foreign_key: 'author_id'
```

Comment:
A Poll does not have an author. It is always the Government.
Please remove this line.

Reply:
When removed, polls show view raises an error requiring "author_id" field

Reply:
We need to remove this as well as the `author_id` column, and find any commentable.autor usage and put a `unless  @comment.commentable.is_a?(Poll)` check after it